spa/src/hooks/usePortalContributorImpact.ts

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query';
+import { getContributorImpactEndpoint } from 'ajax/endpoints';
+import axios from 'ajax/portal-axios';
+
+export interface PortalImpact {
+  /**
+   * Total payments a Contributor made (= total_payments - total_refunded).
```

Comment:
```suggestion
   * Total amount contributed in cents (= total_payments - total_refunded).
```

Believe this is the intent here--we should clarify unit and that this is an amount, not a count (e.g. 12300 contributed, not 12 total contributions).